examples/remote_control_cozmo.py

```diff
@@ -444,6 +444,7 @@ def handle_index_page():
                         <h3>General:</h3>
                         <b>Shift</b> : Hold to Move Faster (Driving, Head and Lift)<br>
                         <b>Alt</b> : Hold to Move Slower (Driving, Head and Lift)<br>
+                        <b>L</b> : Toggle IR HeadLight: <button id="headLightId" onClick=onHeadLightButtonClicked(this) style="font-size: 14px">Default</button><br>
```

Comment:
"HeadLight" does not typically have an internal uppercase L (so normally it is "Headlight"). Can we make the internal uppercase L be lowercase throughout or at least everywhere the user will see it (i.e., text, code)?